build.sbt

```diff
@@ -94,11 +94,19 @@ lazy val commonSettings = Seq(
   )
 )
 
+ThisBuild / organization := "com.banno"
+Compile / scalacOptions ++= Seq(
+  "-groups",
+  "-sourcepath",
+  (LocalRootProject / baseDirectory).value.getAbsolutePath,
+  "-doc-source-url",
+  "https://github.com/banno/cosmos4s/blob/v" + version.value + "€{FILE_PATH}.scala"
```

Comment:
Is the `€` correct?

Reply:
Yes it is, but I can not find the explanation in the documentation.